dhis-2/dhis-services/dhis-service-setting/src/main/java/org/hisp/dhis/setting/SettingKey.java

```diff
@@ -59,7 +59,7 @@
  */
 public enum SettingKey {
   UI_LOCALE("keyUiLocale", LocaleManager.DEFAULT_LOCALE, Locale.class),
-  DB_LOCALE("keyDbLocale", Locale.class),
+  DB_LOCALE("keyDbLocale", LocaleManager.DEFAULT_LOCALE, Locale.class),
```

Comment:
This fix does not satisfy a conner case:
- An instance has DB_Locale = null. User enter value in Chinese.  The expected behavior is the raw value ( in Chinese ) should be returned. However with this fix, the DB_Locale is English so the return value will be the translated value in English, which is taken from `translations` jsonb column. 
- If the English translation value does not exist then the raw value will be returned.

Please have a look at [this code](https://github.com/dhis2/dhis2-core/blob/b2f0b6375c8c7ce21264c4300015e4bca0292939/dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/BaseIdentifiableObject.java#L336)